test-integration/helpers/task-plan-builder.coffee

```diff
@@ -285,5 +250,30 @@ class ReadingBuilder extends TestHelper
       when 'CANCEL' then @cancel()
       when 'DELETE' then @delete()
 
+  selectNumberOfExercises: (numExercises=4) ->
+    @_exerciseSelector.waitUntilLoaded()
+    @_exerciseSelector.selectNumberOfExercises(numExercises)
+
+  verifySelectedExercises: (numExercises) ->
+    @el.selectedExercises.findElements().then (els) ->
+      expect(numExercises).to.be.equal(els.length)
```

Comment:
I agree that i would expect the `expect` to be in the spec...i might expect to see this in there though, but it's not really needed:

```coffee
countSelectedExercises: ->
   @el.selectedExercises.findElements().then (els) ->
     els.length
```

then the spec might be
```coffee
   # stuffs
   @taskPlanBuilder.countSelectedExercises().then (numOfSelected) ->
      expect(numExercises).to.be.equal(numOfSelected)
```